Mathlib/Topology/Algebra/Module/Dual.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2024 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import Mathlib.LinearAlgebra.SesquilinearForm
+import Mathlib.Topology.Algebra.Module.Basic
+
+/-!
+# Topological dual
+
+## Main definitions
+
+* `Dual`: abbreviation for `E →L[𝕜] 𝕜`.
+* `dualPairing`: the canonical pairing `Dual 𝕜 E →ₗ[𝕜] E →ₗ[𝕜] 𝕜`.
+
+## Main statements
+
+* `dualPairing_separatingLeft`: the dual pairing is always left separating
+
+-/
+
+noncomputable section
+
+open Filter Topology
+
+variable {𝕜 E F ι : Type*}
+
+section CommSemiring
+
+variable [CommSemiring 𝕜] [TopologicalSpace 𝕜] [ContinuousAdd 𝕜] [AddCommMonoid E]
+    [Module 𝕜 E] [TopologicalSpace E] [ContinuousConstSMul 𝕜 𝕜]
+
+variable (𝕜 E) in
+/-- The topological dual of a topological vector space `E`. -/
+abbrev Dual : Type _ := E →L[𝕜] 𝕜
```

Comment:
Is there a consensus that `Dual` with this meaning can be in the root namespace? Currently it is in `NormedSpace.Dual`. It seems quite clear indeed that `NormedSpace` namespace is not the best choice anymore (not general enough), but in my opinion this PR's proposal to change directly to the root namespace requires a wide agreement, because dual might be standard terminology in other contexts as well. (I would personally be in favor, though.)

Reply:
I think the topological dual is more meaningful than the algebraic one. However, it would be the least confusing if both are not in the root namespace and have an indication which one they are.